google/ggrc-core
# GGRC-6365 Replace class listeners with canjs events

# Issue description

Unobvious event handling in some places.

# Steps to test the changes

1. Bulk-update btn && object-bulk-update
2. Open My tasks tab
3. Click on three-dots menu and click 'Bulk update'
4. Expected: Bulk update modal opened
5. Choose tasks and click update
6. Expected: Update btn works
5. Object mapper
6. Try to map any object to another object.
7. Expected: Close modal btn and map btn works, mapping works
8. Object generator
9. Click 'Autogenerate' btn on audit summary page
10. Try to generate assessment/close modal.
11. Expected: Close modal btn, Generate assessment btn, Autogenerate btn works
12. Click 'Create template' btn on audit summary page
13. Try to clone template
14. Expected: It works
15. ggrc-gdrive-folder-picker
16. Click 'Assign folder' and add folder on object where it possible
17. Remove folder
18. Expected: It works
19. unarchive btn
20. Archive any audit
21. Try to unarchive this
22. Expected: Unarchive btn works.


# Solution description

Added `el:on:click` listeners for stache files where it possible, changed components logic related to this event listeners. Logic is connected to list items, event's propagations, controllers and some elements can't be changed because it affected performance or events behaviour.

# Sanity checklist




# PR Review checklist
